modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -346,6 +364,37 @@ public void testPostDataLayoutContext() throws Exception {
 		Assert.assertEquals(200, httpResponse.getStatusCode());
 	}
 
+	@Test
+	public void testPutDataDefinitionDataLayout() throws Exception {
+		DataLayout postDataLayout = testPutDataLayout_addDataLayout();
```

Comment:
can you call super before everything? thx

Reply:
Added to `testPostDataDefinitionDataLayout_addDataLayout` in the previous block and to `dataLayoutResource`, but just to be sure, `testPutDataLayout_addDataLayout` is not in the parent class.

Reply:
`testPutDataDefinitionDataLayout` is a new method and has no corresponding in the parent class :thinking:

Reply:
Hum.. that way we should fit this test case in a existing test in the parent class

Reply:
I'll check if there is a suitable test in the parent class, then

Reply:
Found `testPutDataLayout` that I believe can be used. What do you think?

Reply:
Agreed!